modules/common-fs2/src/test/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/config/ConfigFileSpec.scala

```diff
@@ -15,21 +15,15 @@ package com.snowplowanalytics.snowplow.enrich.common.fs2.config
 import java.net.URI
 import java.util.UUID
 import java.nio.file.Paths
-
 import scala.concurrent.duration._
-
 import cats.syntax.either._
 import cats.data.NonEmptyList
 import cats.effect.IO
-
 import cats.effect.testing.specs2.CatsIO
-
+import com.snowplowanalytics.snowplow.enrich.common.SpecHelpers._
```

Comment:
We usually put Snowplow packages at the bottom of the import

Reply:
Have tried to follow this convention too.